zeplin/Shared/NetworkProvider.swift

```diff
@@ -8,18 +8,18 @@
 
 import Foundation
 import UIKit
-import ios_toolkit
+import Toolkit
 import RxSwift
 import KeychainSwift
-import Alamofire
+import Moya
+import RxMoya
+import RxMoya_ModelMapper
 
 class NetworkProvider {
     static let shared = NetworkProvider()
     
-    let manager = SessionManager.default
-    
-    let baseUrl = "https://api.relevantfruit.com/v1/zeplin/"
-    
+    let provider = MoyaProvider<ZeplinService>()
+        
     private var keychain = KeychainSwift()
```

Comment:
NetworkService.swift için yazdığım KeychainService yorumuna dayanarak burayı silebilirsin. Keychain'e herhangi bir şey yazmayı ve ordan bir şey okumayı tek bir servis üzerinden yapmalısın. Kod içinde `KeychainSwift()` ler olmamalı.